src/config/schema/split-document-schema-validator.ts

```diff
@@ -0,0 +1,20 @@
+import Ajv, { JSONSchemaType } from 'ajv';
+import { SplitDocumentConfig } from '../interfaces';
+
+const schemaValidator = new Ajv({
+  allErrors: true,
+});
+
+const SplitDocumentSchema: JSONSchemaType<SplitDocumentConfig> = {
+  type: 'object',
+  properties: {
+    path: { type: 'string' },
+    header: { type: 'boolean' },
+    linesPerFile: { type: 'number' },
+    filesPerFolder: { type: 'number' },
```

Comment:
should we validate the configuration here?
```suggestion
    linesPerFile: { type: 'number', minimum: 1 },
    filesPerFolder: { type: 'number', minimum: 1 },
```